configs/javascript.js

```diff
@@ -4,10 +4,17 @@ module.exports = {
     browser: true,
     node: true,
   },
-  extends: ['eslint:recommended', 'prettier'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
```

Comment:
I'm not confident that it's sane to enable this ruleset for plain javascript projects.

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/eslint-recommended.ts says the following:

```
/**
 * This is a compatibility ruleset that:
 * - disables rules from eslint:recommended which are already handled by TypeScript.
 * - enables rules that make sense due to TS's typechecking / transpilation.
 */
```

I'm assuming it means the typescript compiler here. But projects that don't use the typescript compiler will miss out on some valid rules. It also enables some rules that could be unwanted for plain javascript projects.